src/polyfills/details/details.js

```diff
@@ -107,7 +107,9 @@ $document.on( "click keydown toggle." + componentName, selector, function( event
 		( event.type === "toggle" && event.namespace === componentName ) ) ) {
 		details = currentTarget.parentNode;
 		details.open = !details.open;
-	} else if ( which === 13 || which === 32 ) {
+
+	// Prevent double handling of enter / spacebar if within a menu (handled by menu.js)
+	} else if ( ( which === 13 || which === 32 ) && currentTarget.getAttribute( "role" ) !== "menuitem" ) {
```

Comment:
Is there another way to handle this? The polyfill shouldn't really care about other WET plug-ins

Reply:
I was able to fix it by adding `event.stopImmediatePropagation()` in menu.js. This depends on the menu.js handler being called before the details.js handler, which it is. Is this an acceptable solution?